### ISSUE TITLE
[NC 13.0.0] CSS: actions.creatable overwrites actions.hidden

### ISSUE THREAD

**Issue by Cybso** [nextcloud/server#8238]
.actions.creatable with 'display: flex' in apps/files/css/files.scss overwrites .actions.hidden with 'display: none', causing the "new" button to be visible when it should not be.

```
.actions.hidden { display: none; }

.actions.creatable {
    position: relative;
    display: flex;
    flex: 1 1;
    .button:not(:last-child) {
        margin-right: 3px;
    }
}
```

Additionally, the gallery button is within the actions div, so it would disappear if this is fixed and the user has no UPDATE permission.

**Comment by MorrisJobke** [comment]
cc @skjnldsv

**Comment by skjnldsv** [comment]
also cc @danxuliu since he fixed a lot on the controls and gallery! :)

**Comment by danxuliu** [comment]
The mentioned issues should have been fixed by #8595 and nextcloud/gallery#402, which were included in Nextcloud 13.0.1.

@Cybso Could you check again on that version? Thanks!
